core/store/tx_manager.go

```diff
@@ -714,23 +721,61 @@ func (txm *EthTxManager) handleSafe(
 	return nil
 }
 
-// bumpGas creates a new transaction attempt with an increased gas cost
+// BumpGasByIncrement returns a new gas price increased by the larger of either
+// a percentage bump or a fixed size bump
+func (txm *EthTxManager) BumpGasByIncrement(originalGasPrice *big.Int) *big.Int {
+	// Similar logic is used in geth
+	// See: https://github.com/ethereum/go-ethereum/blob/8d7aa9078f8a94c2c10b1d11e04242df0ea91e5b/core/tx_list.go#L255
+	// And: https://github.com/ethereum/go-ethereum/blob/8d7aa9078f8a94c2c10b1d11e04242df0ea91e5b/core/tx_pool.go#L171
+	minimumGasBumpByPercentage := new(big.Int).Div(
```

Comment:
Would it make sense to also check the gas price against the geth's [`client.SuggestGasPrice`](https://github.com/ethereum/go-ethereum/blob/master/ethclient/ethclient.go#L485) value? If that's larger, we might save the node quite a bit of work by just jumping straight there.

Reply:
> by just jumping straight there

(Or perhaps to some large fraction of it, if we want to be conservative.)

Reply:
Good idea... that's probably a question for @se3000

Reply:
Steve says:

```
it does not price very aggresively
11:35
we want a price that will get us in the next block
11:35
or something close to it
11:36
our reliance on the external gas pricing sites is not great
```

For this reason, I think we should leave this to a later PR.